tests/unit/helpers/assert-helpers-test.js

```diff
@@ -120,3 +120,105 @@ test('assert.classNotIncludes / assert.classNotContains', function(assert) {
   assert.classNotIncludes(this.$('div'), 'bar', 'it works');
   assert.classNotContains(this.$('div'), 'bar', 'it works');
 });
+
+test('assert.tableContains', function(assert) {
+  assert.expect(1);
+
+  this.render(hbs`
+    <table>
+      <tbody>
+        <tr>
+          <td>1</td>
+          <td>2</td>
+        </tr>
+        <tr>
+          <td>3</td>
+          <td></td>
+        </tr>
+      </tbody>
+    </table>
+  `);
+
+  assert.tableContains(this.$('table:eq(0)'), {
+    body: [
+      [1,2],
+      [3]
+    ]
+  });
+});
+
+test('assert.tableContains - with a null cell', function(assert) {
+  assert.expect(1);
+
+  this.render(hbs`
+    <table>
+      <tbody>
+        <tr>
+          <td>1</td>
+          <td>2</td>
+        </tr>
+        <tr>
+          <td>3</td>
+          <td>4</td>
+        </tr>
+      </tbody>
+    </table>
+  `);
+
+  assert.tableContains(this.$('table:eq(0)'), {
+    body: [
+      [1,2],
+      [null, 4]
```

Comment:
Oh I think I understand. So, if you only want to check certain values are present, you can just put null for the others?